nav2_amcl/src/amcl_node.cpp

```diff
@@ -845,18 +845,18 @@ AmclNode::laserReceived(sensor_msgs::msg::LaserScan::ConstSharedPtr laser_scan)
       // p.covariance[6*5+5] = hyps[max_weight_hyp].pf_pose_cov.m[2][2];
       p.pose.covariance[6 * 5 + 5] = set->cov.m[2][2];
 
-      /*
-         printf("cov:\n");
-         for(int i=0; i<6; i++)
-         {
-         for(int j=0; j<6; j++)
-         printf("%6.3f ", p.covariance[6*i+j]);
-         puts("");
-         }
-       */
+      float temp = 0.0;
+      for (int i = 0; i < static_cast<int>(p.pose.covariance.size()); i++) {
+        temp += p.pose.covariance[i];
+      }
+      temp += p.pose.pose.position.x + p.pose.pose.position.y;
+      if (!std::isnan(temp)) {
+        RCLCPP_INFO(get_logger(), "AmclNode publishing pose");
```

Comment:
This should probably be a debug print, this would get old really fast when trying to debug other issues with a floor of these messages.